Microsoft.DotNet.Interactive.Tests/PackageRestoreContextTests.cs

```diff
@@ -119,21 +119,26 @@ public async Task Can_get_path_to_nuget_packaged_assembly()
 
             var path = packageReference.AssemblyPaths.Single();
 
-            path.FullName
-                .ToLower()
-                .Should()
-                .EndWith("fluentassertions" + Path.DirectorySeparatorChar +
-                         "5.7.0" + Path.DirectorySeparatorChar +
-                         "lib" + Path.DirectorySeparatorChar +
-                         "netcoreapp2.0" + Path.DirectorySeparatorChar  +
-                         "fluentassertions.dll");
+            // path is a string similar to:
+            /// c:/users/someuser/.nuget/packages/fluentassertions/5.7.0/netcoreapp2.0/fluentassertions.dll
+            var name = path.Name;
+            var tfm = path.Directory.Name;
+            var reflib = path.Directory.Parent.Name;
+            var version = path.Directory.Parent.Parent.Name;
+            var packageName = path.Directory.Parent.Parent.Parent.Parent.Name;
+
+            name.ToLower().Should().Equals("fluentassertions.dll");
```

Comment:
If you want to use multiple assertions in a single test, it's nice to see all of the failures rather than just the first one.

You run all of the assertions before any exception is thrown by doing this before you start asserting:

```csharp
using var _ = new AssertionScope();
```